RELEASE/scripts/sl_ascend/sl_picky.ash

```diff
@@ -74,6 +74,16 @@ void picky_startAscension()
 
 boolean picky_buyskills()
 {
+	// TODO: Torso Awaregness is worth obtaining in other cases too.
+	if(my_meat() >= 10000
+	   && (my_class() == $class[Vampyre])
+	   && gnomads_available()
+	   && (!have_skill($skill[Torso Awaregness]))
+	   && (item_amount($item[January\'s Garbage Tote]) != 0)
+	   && (!is_unrestricted($item[January\'s Garbage Tote])))
```

Comment:
why `!is_unrestricted` instead of `is_unrestricted`?

Reply:
Nice catch! It's totally wrong and is fixed in a later commit in the stack.

Reply:
never mind you fixed that in a later commit